quarkchain/cluster/tests/test_utils.py

```diff
@@ -274,11 +291,12 @@ def contract_creation_tx(
 
 
 class Cluster:
-    def __init__(self, master, slave_list, network, peer):
+    def __init__(self, master, slave_list, network, peer, grpc_server):
```

Comment:
as mentioned before, we don't need to put grpc server into the cluster for unit testing